<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>861</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1184</width>
    <height>641</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{
background: #eaebed;
}
*{
color: #232134;
box-sizing: border-box;
}
#Start, #upload{
	background: #5e96fc;
	border-radius: 20px;
	transition-duration: 1s;
	cursor: pointer;
	color: white;
	font-weight: 700;
}
#Start:hover{
background: #538efc;
}
#upload:hover{
background: #538efc;
}
.QSpinBox, QDoubleSpinBox{
max-width: 60px;
border: none;
border-radius: 20px;
outline: none;
background: white;
border-bottom: 2px solid #5e96fc;
padding: 4px;
}
.QFrame{
background:white;
border-radius: 20px;


}
.QGroupBox{
border: none;
font-size: 16px;
font-weight: 700;
}
</string>
  </property>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>690</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>РАСЧЕТ РЛИ</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>711</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Коэффициент передискретизации: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Kss">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Дискретность по долготе, м: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dxsint">
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Дискретность по широте, м: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dysint">
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Степень яркостного изображения: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="StepBright">
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Число точек по долготе: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Nxsint">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>201</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Число точек по широте: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Nysint">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>201</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="upload">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>ВЫБРАТЬ CONSORT ФАЙЛ</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Путь:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Path">
       <property name="styleSheet">
        <string notr="true">color: #7b7c88</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="file_name_label">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Файл:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="File_name">
       <property name="styleSheet">
        <string notr="true">color: #7b7c88</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>400</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Время синтезирования, c</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Tsint">
         <property name="value">
          <double>0.400000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Задержка момента построения РЛИ-1, с</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="tauRli"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Интервал времени между РЛИ-1,-2. с</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="t_r_w">
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>1471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>9999999</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #5e96fc;
color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Построение радиолокационного изображения в маршрутном режиме</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>560</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Однопроходная интерферометрия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="FlagViewSignal">
      <property name="text">
       <string>Отображать сигналы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="GPU">
      <property name="text">
       <string>Аппаратное ускорение (GPU)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="FlagWriteRli">
      <property name="text">
       <string>Сохранить результат</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="TypeWinDn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border-radius: 20px;</string>
   </property>
   <property name="title">
    <string>Весовая функция по наклонной дальности</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="IMAGE1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>media/img-HUbuKO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Dn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>косинус</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>косинус квадрат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хемминга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хеммига (третья степень)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хеммига (четвертая степень)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кайзера-Бесселя, alfa=2.7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кайзера-Бесселя, alfa=3.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кайзера-Бесселя, alfa=3.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Бэкмана-Херриса</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TypeWinDp">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border-radius: 20px;</string>
   </property>
   <property name="title">
    <string>Весовая функция по поперечной дальности</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="IMAGE2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>media/8.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Dp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>косинус</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>косинус квадрат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хемминга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хеммига (третья степень)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хеммига (четвертая степень)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кайзера-Бесселя, alfa=2.7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кайзера-Бесселя, alfa=3.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кайзера-Бесселя, alfa=3.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Бэкмана-Херриса</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowserModalDate">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>230</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>721</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QTextBrowser" name="Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;
color:white;
font: 9pt &quot;SimSun-ExtB&quot;;
border: 2px solid white;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="RowsAndCulCount">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>230</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>400</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим съемки</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>124</width>
      <height>114</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Stripmap2">
       <property name="text">
        <string>Маршрутный
 скошенный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Detailmode">
       <property name="text">
        <string>Детальный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Stripmap">
       <property name="text">
        <string>Маршрутный</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame_8</zorder>
  <zorder>frame_7</zorder>
  <zorder>frame</zorder>
  <zorder>Start</zorder>
  <zorder>frame_2</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>TypeWinDn</zorder>
  <zorder>TypeWinDp</zorder>
  <zorder>textBrowserModalDate</zorder>
  <zorder>groupBox</zorder>
  <zorder>RowsAndCulCount</zorder>
  <zorder>frame_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
